arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -1745,6 +1996,31 @@
         ],
         "parameters": [
           {
+            "name": "deviceName",
```

Comment:
Done. Moved 'deviceName' as a global parameter and at all places of use, added reference to it.